src/app/blogs/[slug]/page.tsx

```
@@ -26,13 +78,53 @@ export default async function BlogDetailPage({
+		},
+		keywords: blogData.tags?.join(", ") || "",
+		articleSection: blogData.blogSlugCategory || "General",
+	};
+
 	return (
-		<div className="relative min-h-screen">
+		<>
+			<Script
+				id="article-schema"
+				type="application/ld+json"
+				dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)